tests/snippets/bytearray.py

```diff
@@ -71,3 +71,21 @@
 a.append(100)
 assert len(a) == 6
 assert a.pop() == 100
+
+a = bytearray(b'hey there, what is up?')
+assert(a.count(b'h', 3))
+assert(a.count(bytearray(b'h'), 3))
+assert(a.count(104, 3))
+try:
```

Comment:
For this type of contraption, we have this:
```python
from testutils import assertRaises

with assertRaises(TypeError):
    raise TypeError  here
```

Reply:
Seems about right, resolved :)